starter-code/css/main.css

```diff
@@ -4,7 +4,176 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
-body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+//*Clearfix
+.group::after {
+     content: ".";
+     display: block;
+     height: 0;
+     clear: both;
+}
+
+/* Hides from IE-mac \*/
+//* html .clearfix {height: 1%;}
+.clearfix {display: block;}
+/* End hide from IE-mac */
+
+h1, body, html, main {
   margin: 0;
 }
+
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+body {
+font-family: Helvetica, Arial, sans-serif;
+width: 100%;
+}
+
+#top-section {
```

Comment:
Try to name your elements in a way that you or someone else can come back to your code months later and understand exactly what every class name is referring to. In the case of this div, you could name it something like 'blue-bg'.